README.md

```diff
@@ -152,7 +152,7 @@
 | 81.  | Rabin Karp | [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Pattern-Matching/rabin_karp.cpp), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Pattern-Matching/rabin_karp.py) | Pattern Matching |
 | 82.  | Z - Algorithm | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Pattern-Matching/Z_Algorithm.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Pattern-Matching/Z_Algorithm.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Pattern-Matching/Z_Algorithm.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Pattern-Matching/Z_Algorithm.py) | Pattern Matching |
 | 83.  | String Permutation | [C](https://github.com/Kumar-laxmi/Algorithms/blob/main/C/Recursion/Permutations.c), [C++](https://github.com/Kumar-laxmi/Algorithms/blob/main/C%2B%2B/Recursion/Permutations.cpp), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Recursion/Permutations.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Recursion/Permutations.py) | Recursion |
-| 84.  | Tower of Hanoi | [C](), [C++]() | Recursion |
+| 84.  | Tower of Hanoi | [C](), [C++](), [Java](https://github.com/Kumar-laxmi/Algorithms/blob/main/Java/Recursion/TowerOfHanoi.java), [Python](https://github.com/Kumar-laxmi/Algorithms/blob/main/Python/Recursion/TowerOfHanoi.py) | Recursion |
```

Comment:
Please do not make any changes to README.md

Reply:
ohhkay sorry... resolved it now...